gsplat/compression/compress.py

```diff
@@ -0,0 +1,340 @@
+import os
+from typing import Any
+import json
+import torch
+from torch import Tensor
+import torch.nn.functional as F
+from torchpq.clustering import KMeans
+import numpy as np
+import imageio
+
+from plas import sort_with_plas
```

Comment:
As of now the only dependency of gsplat is torch. Personally I'm a fun of minimal dependencies for easy maintainess. Since we are not heavily depends on them, maybe it makes sense to push them into `compress_splats` function with an try expect, and add a `.. warning` in the func doc to tell users how to install them.

btw, just curious, if your env does not have them installed, would it fail to run `from gsplat import rasterization`?

Reply:
Moved kmeans and plas into their respective functions so you shouldn't run into import errors unless you use the --compress flag.